### PR TITLE
Adding SECURITY.md file

### PR DESCRIPTION
**Pull request by gauntface**
This change adds a default security policy for Google repos.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+To report a security issue, please use http://g.co/vulnz. We use
```
Comment (annabellegoth2boss) [inline SECURITY.md:1]: Change to http**s** on lines 1 and 2

**Review by annebdh — CHANGES_REQUESTED** [review]

**Review by annebdh — APPROVED** [review]

**Comment by 1Elle1111** [comment]
Merged

File: SECURITY.md (hunk 1 above)
Comment (1Elle1111) [reply SECURITY.md]: Hi

**Review by 1Elle1111 — COMMENTED** [review]
Hi

**Pull request merged**